src/components/productList/utils/rating-filter.js

```diff
@@ -0,0 +1,7 @@
+export default function filterByRating(allProducts, rating) {
+  let products = [...allProducts];
+  if (rating === 0) {
+    return products;
+  }
+  return products.filter((product) => product.rating === Number(rating));
```

Comment:
- I dont think this is the correct approach for filtering by rating. If you go on any ecom websites, they allow you to filter based on rating & above. So if I select 3 it will show products for 3 star and above. You can have a look at Amazon for inspiration. 
- Also, if you update, please change the label on top of the slider accordingly.

Reply:
Yeah Definitely will do the required changes.